From 2cd507407f3b5f3f947eb06a23032329e3532684 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:13:18 -0400
Subject: [powerpc] fix typo 'CONFIG_PPC_CPU'

Message-id: <1410545655-205645-570-git-send-email-dzickus@redhat.com>
Patchwork-id: 94517
O-Subject: [RHEL7 PATCH 569/626] powerpc: fix typo 'CONFIG_PPC_CPU'
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit b69a1da94f3d1589d1942b5d1b384d8cfaac4500
Author: Paul Bolle <pebolle@tiscali.nl>
Date:   Tue May 20 21:59:42 2014 +0200

    powerpc: fix typo 'CONFIG_PPC_CPU'

    Commit cd64d1697cf0 ("powerpc: mtmsrd not defined") added a check for
    CONFIG_PPC_CPU were a check for CONFIG_PPC_FPU was clearly intended.

    Fixes: cd64d1697cf0 ("powerpc: mtmsrd not defined")
    Signed-off-by: Paul Bolle <pebolle@tiscali.nl>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/lib/sstep.c b/arch/powerpc/lib/sstep.c
index a89ff3e..8c4ae90 100644
--- a/arch/powerpc/lib/sstep.c
+++ b/arch/powerpc/lib/sstep.c
@@ -1470,7 +1470,7 @@ int __kprobes emulate_step(struct pt_regs *regs, unsigned int instr)
     regs->gpr[rd] = byterev_4(val);
    goto ldst_done;
 
-#ifdef CONFIG_PPC_CPU
+#ifdef CONFIG_PPC_FPU
   case 535: /* lfsx */
   case 567: /* lfsux */
    if (!(regs->msr & MSR_FP))
-- 
1.7.1